src/frontend/src/components/OnboardingBlock.tsx

```diff
@@ -1,20 +1,58 @@
-import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { useEffect, useState } from 'react';
+import { Box, Grid, Typography, useTheme, TextField } from '@mui/material';
 import EditIcon from '@mui/icons-material/Edit';
 import { Organization } from 'shared';
+import NERFormModal from './NERFormModal';
+import { useForm } from 'react-hook-form';
+import { useSetOnboardingText } from '../hooks/organizations.hooks';
+import { useToast } from '../hooks/toasts.hooks';
 
 interface OnboardingBlockProps {
   organization: Organization;
   isAdmin?: boolean;
 }
 
+interface OnboardingTextFormData {
+  onboardingText: string;
+}
+
 const OnboardingBlock: React.FC<OnboardingBlockProps> = ({ organization, isAdmin }) => {
   const theme = useTheme();
+  const toast = useToast();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const setOnboardingText = useSetOnboardingText();
+
+  const { register, handleSubmit, reset } = useForm<OnboardingTextFormData>({
+    defaultValues: {
```

Comment:
the useForm also needs to take in a resolver using the yup.schema check. look at how its done in other ones